### ISSUE TITLE
Automate `valkey-hashes` commit

### ISSUE THREAD

**Issue by vudiep411** [madolson/placeholderkv#1396]
### The problem/use-case that the feature addresses

Currently we're manually committing the hashes into [valkey-hashes](https://github.com/valkey-io/valkey-hashes) every new release. 

### Description of the feature
We could add an automated commit the new hashes step into `build-release-packages`. When releasing a new version the workflow will run the following steps:
1. Build the binary files and upload them into S3 bucket to serve [valkey.io](https://valkey.io/download/) website.
2. **Run additional workflow that commit the new hash into [valkey-hashes](https://github.com/valkey-io/valkey-hashes) repo for checksum**

### Prerequisite
Create a PAT that allow write access to [valkey-hashes](https://github.com/valkey-io/valkey-hashes).

### Additional Information
Here is the proof of concept for the automation
[Code](https://github.com/Autxmaton/valkey/blob/unstable/.github/workflows/build-release-packages.yml#L119-L126)

![Screenshot 2024-12-05 at 1 58 45 PM](https://github.com/user-attachments/assets/42284b9b-c1ff-4557-97c3-26fb38903641)

> The runner made the new commit to this repo

Let me know if it is a good idea.

**Comment by vudiep411** [comment]
@roshkhatri

**Comment by roshkhatri** [comment]
Yes, this is what I was discussing with you the other day.

**Comment by roshkhatri** [comment]
@valkey-io/core-team PTAL, we would also create future automation after this to commit on valkey-containers and eventually also update the links for the binaries download pages for valkey-website.

**Comment by roshkhatri** [comment]
Also, For this and future automations, Can we setup a bot account which will have the PAT for the repos like valkey-hashes and valkey-container.

Once the release is complete on the main repo, it will trigger the bot to make the commits on valkey-hashes and valkey containers.

Keeping workflows for the commits on other repos in the main repo does not make sense to me.

I think we should centralize these in a repository called as `valkey-automation-bot` in a `valkey-bot-user` which will house scripts for all the commits and automations for the release, triggered by the release workflow on the `valkey` repo.

I think this needs an issue by itself. I will create it

**Comment by vudiep411** [comment]
I agree with this design. We can set a trigger workflow step in this main valkey-repo that will trigger the bot automation and it will make commits in the appropriate repos.
